manifests/rbac/cluster-role.yaml

```
@@ -0,0 +1,12 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+ name: external-mdns
+rules:
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["list", "watch"]
+- apiGroups: ["extensions","networking.k8s.io"]
```

```suggestion
- apiGroups: ["extensions","networking.k8s.io"]
```

What's the reason for adding the `extensions` API group?

---

Uhm I have used manifest I had around from other projects if you copied it from mine and then copied it in your PR I just did the same thing back :D

---

Oh, right. My mistake. 🤦‍♂️ It has been in the repo for quite some time.

I realize this was missing in the first commit in this PR, but then you re-added in when pulling in #31.